lib/auth.js

```diff
@@ -48,6 +55,66 @@ Auth.prototype.bearer = function (bearer, sendImmediately) {
   }
 }
 
+
+Auth.prototype.negotiate = function(request, authHeader, sendImmediately) {
+  var self = this
+
+  if (kerberos === undefined) {
```

Comment:
That's probably not needed, as this method looks public, but in fact it's used through internal `_auth` instance, and you are checking for undefined inside the `onRequest` method.